.env.example

```diff
@@ -0,0 +1,6 @@
+VERSE_MASTER_NODE_URL=
+VERSE_URL=
+VERSE_READ_NODE_URL=http://localhost:8545
+BLOCK_NUMBER_CACHE_EXPIRE_SEC=
+DATASTORE=
+NODE_SOCKET=ws://[::]:8546
```

Comment:
@Sotatek-JohnnyNguyen 
`.env.example` file has been added, but how is it intended to be used?
It appears that `dotenv` is not included in the package.json.
Will you use the `source .env` command?

And it would be helpful if you could also update the `README.md` file to reflect the current situation.

Although it is unrelated to this change, I believe that `$ npm build` in the README.md file should be corrected to `$ npm run build`.

Reply:
First: Nestjs provide configModule that exposes a configService which loads the appropriate `.env` file, so we dont need dotenv for now
Second: the `.env.example` is like an example to help other devs know which configuration variables they have to add to `.env` file in order to run the service

Reply:
I will update README.me file now. Also yes both commands `$ npm build` and `$ npm run build` are doing the same action

Reply:
@Sotatek-JohnnyNguyen 
Okay. Thanks for replying.
By the way, what do you think about including the environment variable `PORT=`, which is currently written to be exported in the README.md, in the .env.example file?
If there are no issues, I feel it would be good to add it.

Reply:
sure